### ISSUE TITLE
ROS2 Support

### ISSUE THREAD

**Issue by chris-ferone** [OxfordTechnicalSolutions/oxts_ros2_driver#1]
Hi, 

Is Foxy the only distribution currently supported? If so, do you know when newer ROS2 distributions will be supported? Thanks!

-Chris

**Comment by SSouliman** [comment]
Hi Chris,

Yes, Foxy is the only distribution supported at the moment. We plan to support Galactic soon, and then to follow the release cycle of ROS2, starting with Humble due in May2022.

Please let us know if you have any more questions.

Regards,

**Comment by eansell-oxts** [comment]
Hi Chris,

Currently, Foxy is the only distribution officially supported, though we do intend to support [whichever versions of ROS 2 are supported officially](https://docs.ros.org/en/foxy/Releases.html), currently Foxy and Galactic. You can run the current version of the driver on Galactic as is, but you have to compile it with Galactic's setup script sourced. Though this should work fine, please bear in mind however that this is untested.

Regards.

**Comment by chris-ferone** [comment]
Thanks for update! It turns out other requirements are forcing us to use Foxy anyway, so its not a problem. Thanks again.

**Comment by chris-ferone** [comment]
Hi Oxts, 

I'm having trouble trying to deploy the driver using ros2 foxy. After running `ros2 launch oxts run.py` I only see "Waiting for INS config information..." I have confirmed the following already: 

1. I've specified the proper IP address of the RT3000 (unit_ip rosparam) in the config.yaml before building.
2. I've confirmed I can ping the RT3000 successfully
3. Using NavDisplay, I've confirmed the RT3000 is reporting a RTK integer fix and the lat,long, alt reported looks correct. 

Any suggestions on how can I further troubleshoot this issue? Thanks!
